docs/.vuepress/config.js

```diff
@@ -28,6 +28,12 @@ module.exports = {
     repo: 'vuejs/vuepress',
     editLinks: true,
     docsDir: 'docs',
+    algolia: {
+      appId: 'BH4D9OD16A',
```

Comment:
Should the appId and apiKey be committed as part of the repo?

Reply:
Hmmm... At thePR summary, I have said:

> Need to clean config ... before merged.